### PR TITLE
Slash command localization support

### PR DESCRIPTION
**Pull request by VelvetThePanda**
Adds localization support to slash commands. This is currently under beta testing and I've yet to even test it myself.

### CODE REVIEW

File: DSharpPlus.SlashCommands/Localization.cs [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+namespace DSharpPlus.SlashCommands
+{
+    /// <summary>
+    /// Supported locals for slash command localizations.
+    /// </summary>
+    public enum Localization
+    {
+        AmericanEnglish,
+        BritishEnglish,
+        PeurtoRicoEnglish,
```
Comment (baruchvlz) [inline DSharpPlus.SlashCommands/Localization.cs:36]:
```suggestion
        PuertoRicoEnglish,
```

**Review by kuylar — CHANGES_REQUESTED** [review]

File: DSharpPlus.SlashCommands/Localization.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (VelvetThePanda) [reply DSharpPlus.SlashCommands/Localization.cs:36]: Blame devsnek /ij

**Comment by VelvetThePanda** [comment]
If it breaks we hard-boil the egg and send our sacrifice to Emzi.

**Comment by kuylar** [comment]
I'm alright with that :v:

**Pull request merged**